cmd/ctr/utils.go

```diff
@@ -79,3 +85,68 @@ func waitContainer(events execution.ContainerService_EventsClient, id string, pi
 		}
 	}
 }
+
+func forwardAllSignals(containers execution.ContainerServiceClient, id string) chan os.Signal {
+	sigc := make(chan os.Signal, 128)
+	catchAll(sigc)
+	go func() {
+	Signals:
+		for s := range sigc {
+			var sig syscall.Signal
+			for sigStr, sigN := range signalMap {
```

Comment:
I thought the same , but the docker client does it this way and I thought there was a reason.
https://github.com/moby/moby/blob/master/cli/command/container/tty.go#L86